src/UKHO.S100PermitService.API/Controllers/BaseController.cs

```diff
@@ -1,17 +1,22 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using System.Diagnostics.CodeAnalysis;
+using UKHO.S100PermitService.Common;
 
 namespace UKHO.S100PermitService.API.Controllers
 {
     [ExcludeFromCodeCoverage]
     public abstract class BaseController<T> : ControllerBase
     {
         private readonly IHttpContextAccessor _httpContextAccessor;
-        protected new HttpContext HttpContext => _httpContextAccessor.HttpContext!;
 
         protected BaseController(IHttpContextAccessor httpContextAccessor)
         {
             _httpContextAccessor = httpContextAccessor;
         }
+
+        protected string GetCurrentCorrelationId()
```

Comment:
Just call this method GetCorrelationId()

Reply:
Resolved